ndk-glue/src/lib.rs

```diff
@@ -55,6 +55,7 @@ lazy_static! {
 
 static mut NATIVE_ACTIVITY: Option<NativeActivity> = None;
 
+#[deprecated]
```

Comment:
Not sure if this is a good idea. If it is, at least set a sensible message here (`#[deprecated = "..."]`).

Reply:
will remove